### DISCUSSION TITLE
Random row effect - intercept + coeff ouput

### DISCUSSION THREAD

**Discussion by gooner255** [JenniNiku/gllvm#113] [Q&A] [answered]
Hi Bert,

If my model was a NBinomal with random row row-effect, and I wanted to interpret a couple of significant species-specific coefficients under a given X variable..... is the output a change in counts or a change in the percentage of that species?

For example, if species A had an intercept of 0.514, coefX of +0.078, under conditions of X = 30, then
exp(0.514 + 0.078 * 30) = 17.36

Is this an increase of 17.36 % or an increase in 17 individuals at X=30?
Thanks!

**Comment by BertvanderVeen** [comment, marked as answer]
Hello! 

In my opinion, coefficients in GLM-type models are best interpreted in a relative manner; negative coefficients can be interpreted as decreasing the mean, positive coefficients as increasing the mean. However, if you want a more meaningful interpretation I would suggest looking into resources on GLMs, as there are a lot that will do better in explaining this than I will.

Your question relates to log-linear models generally, so models with (for example) a Poisson distribution are similarly interpreted.

You could read https://www.middleprofessor.com/post/interpreting-coefficients-in-glms/, which is a resource by Gordana Popovic. Or for example: https://www.r-bloggers.com/2018/10/generalized-linear-models-understanding-the-link-function.

**Comment by gooner255** [comment]
Hi Bert!

Thats brilliant! Thanks so much for the help....  I got a little confused looking at sources of GLM interpretation with identity links!
